scripts/remove-default-apps.ps1

```diff
@@ -27,12 +27,12 @@ $apps = @(
     "Microsoft.People"
     "Microsoft.SkypeApp"
     #"Microsoft.Windows.Photos"
-    "Microsoft.WindowsAlarms"
+    #"Microsoft.WindowsAlarms"
     #"Microsoft.WindowsCalculator"
-    "Microsoft.WindowsCamera"
+    #"Microsoft.WindowsCamera"
```

Comment:
Useful for testing laptop's cameras. Efficient program for taking screenshots. It's core application not a bloatware. 
By removing this APP, most of the laptop computers lose ability to take photos and record video.
By removing this, computer loses component functionality as, there is not many apps to replace this one.